src/oed.wppl

```diff
@@ -91,15 +95,14 @@ var EIG = function(args) {
                 var scoreY = m(x,y);
                 factor(scoreY);
             }
-            // TODO: just use AIG() here?
-            var mPosterior = inferM2(function() {
-                var m2Name = mNameSample(), m2 = mFuncs[m2Name];
-                var ll = m2(x,y);
-                factor(ll);
-                return m2Name
-            });
-            var kl = KL(mPosterior, mPrior);
-            return kl;
+
+            return AIG({
```

Comment:
Directive -- we should make AIG accept an optional model prior argument as an optimization, but this is going to look a little different based on some cleanup work I'm doing, so let's table it for now.